client/components/me-alerts/me-alerts.controller.js

```diff
@@ -0,0 +1,24 @@
+class MeAlertsCtrl {
```

Comment:
I thought the same think... MeAlerts don't say anything about the purpose of the controller / page / service, etc.